frontend/ui/data/css_tweaks.lua

```diff
@@ -943,6 +943,15 @@ This only works with footnotes that have specific attributes set by the publishe
     -cr-only-if: -fb2-document;
         -cr-hint: footnote-inpage;
         margin: 0 !important;
+}
+*, autoBoxing {
+    -cr-hint: late;
+    -cr-only-if: inpage-footnote;
+        font-size: 0.8rem !important;
+}
+*, autoBoxing {
+    -cr-hint: late;
+    -cr-only-if: extended-inpage-footnote;
```

Comment:
Can't the first `*, autoBoxing {` content still be held in the `*[role~="doc-rearnote"] {`, to avoid a CSS check ?

Reply:
Sure. I think I was splitting it out because I was already thinking about https://github.com/koreader/koreader/pull/13613 and it needs to be separate there.

Reply:
Reduced this to one extra CSS check with https://github.com/koreader/crengine/pull/620